### PR TITLE
VB-314 Prisoner profile page

### PR DESCRIPTION
**Pull request by tpmcgowan**

### CODE REVIEW

File: server/data/prisonApiClient.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  getOffender(offenderNo: string): Promise<InmateDetail> {
+    return this.restclient.get({
+      path: `/api/offenders/${offenderNo}`,
+    })
+  }
+
+  async getVisitBalances(offenderNo: string): Promise<VisitBalances | null> {
+    let balances: VisitBalances
+    try {
+      balances = await this.restclient.get({
```
Comment (psoleckimoj) [inline server/data/prisonApiClient.ts:24]: Could always `return` here, then change the last `return` on line 34 to just `return null`

Comment (psoleckimoj) [reply server/data/prisonApiClient.ts:24]: Then no need for `balances`

Comment (tpmcgowan) [reply server/data/prisonApiClient.ts:24]: Good suggestion, thanks 👍!

File: assets/sass/components/_prisoner-profile.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  &--RCDR {
+    background-color: govuk-colour("dark-blue");
+  }
+  
+  &--URCU {
+    background-color: govuk-colour("purple");
+  }
+}
```
Comment (psoleckimoj) [inline assets/sass/components/_prisoner-profile.scss:52]: needs a newline 👍

**Review by psoleckimoj — DISMISSED** [review]
LGTM! Just wondering if we just have HTML in that type? It doesn't stop us just having text and may make it less complicated? We're also only using HTML now.

File: assets/sass/components/_prisoner-profile.scss (hunk 2 above)
Comment (tpmcgowan) [reply assets/sass/components/_prisoner-profile.scss:52]: Keep forgetting those 🤦‍♂️!

**Comment by tpmcgowan** [comment]
👍 will try out the type suggestion...

**Comment by tpmcgowan** [comment]
Have added a change to simplify the type for the search result table row items to just have `html` rather than `html` or `text`. Looking in the govuk table macro it seems only difference is whether they're escaped or not so I think it'll be fine given how we're currently using it.

**Review by psoleckimoj — APPROVED** [review]
LGTM :)

**Pull request merged**